tap_bigcommerce/schemas/orders.json

```diff
@@ -427,7 +427,7 @@
                   "$ref": "type-string.json"
                 },
                 "value": {
-                  "$ref": "type-number.json"
+                  "type": ["number", "string", "null"]
```

Comment:
Hi @shikharjaiswal67,

I was wondering about the necessity of maintaining two separate datatypes. The documentation specifically mentions the datatype as `string`. Should we consider eliminating the `number` datatype altogether?

Having both datatypes may lead to the creation of two columns in the destination table, each with a different datatype (number and string). What are your thoughts on this?

Reply:
hi @sgandhi1311 ,

Agreed with you as there is no need for both datatypes. I will create update the PR and let you know.